src/v2/view-builder/views/duo/ChallengeDuoAuthenticatorView.js

```diff
@@ -0,0 +1,35 @@
+import { loc } from 'okta';
+import Duo from 'duo';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+
+  className: 'duo-authenticator-challenge',
+  noButtonBar: true,
+
+  title  () {
+    return loc('oie.duo.verify.title', 'login');
+  },
+
+  postRender: function () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    // TODO Check contextualData.integrationType once we support other types than iframe
+    this.add('<iframe frameborder="0" title="' + this.title() + '"></iframe>');
+    Duo.init({
```

Comment:
Would it make sense to create a separate/util function to initialize Duo, so that it can be reusable ?

Reply:
Are there use-cases where we'd want to reuse this? I'd assume this only needs to get triggered on enroll/challenge - both scoped appropriately.

Reply:
I feel we already have the `Duo` functions in a different library... in this specific case, it happens that enroll and challenge are the same. I thought of adding a parent View with it, and then 2 child views (one enroll and one challenge) that will change the title, but it seem overenginered for a simple view... let me know your thoughts, but I lean towards keeping it as is